pkg/ebpf/events_processor.go

```diff
@@ -329,6 +330,42 @@ func (t *Tracee) processEvent(event *trace.Event) error {
 			return fmt.Errorf("error parsing cgroup_rmdir args: %v", err)
 		}
 		t.containers.CgroupRemove(cgroupId)
+
+	case DetectHookedSyscallsEventID:
```

Comment:
The current name is problematic for the BPF name, but it does suit the user-mode event.
What I think you should do is to create 2 events - 1 for syscalls addresses and 1 for hooked syscalls.
I think it will match the logic of "derived events", because it is an event that is derived from another event.